### DISCUSSION TITLE
NeuralRecon Demo Not working

### DISCUSSION THREAD

**Discussion by SachinBAMP** [zju3dv/NeuralRecon#48] [General]
Hi,

I tried to use demo.py but I couldn't get all the dependencies working.  
python demo.py --cfg ./config/demo.yaml

Error: 
ModuleNotFoundError: No module named 'torchsparse'

The code depends on mit-han-lab/torchsparse. I tried to install this on mac and on windows, but both of them failed.

Mac: Following command didn't work:
pip install --upgrade git+https://github.com/mit-han-lab/torchsparse.git@v1.4.0

Error:
Failed to install wheel for torchsparse

Window: torchsparse failed to install

Cannot open include file: 'google/dense_hash_map': No such file or directory

**Comment by JiamingSuen** [comment]
`torchsparse` only supports Linux, please find a Linux machine to run the demo.
